fenix/app/src/main/AndroidManifest.xml

```diff
@@ -146,6 +146,8 @@
                     android:host="urls_bookmarks"/>
                 <data android:scheme="${deepLinkScheme}"
                     android:host="urls_history"/>
+                <data android:scheme="${deepLinkScheme}"
+                    android:host="test_deferred_deep_link"/>
```

Comment:
I remain a bit skeptical about this, so just trying to validate assumptions: as I understand it, you are hoping that the Play Store will - *after* the user installs the app - initiate an intent with `test_deferred_deep_link` as the `host` part of the URI, thus triggering Fenix to open & capture it with this IntentFilter. Did I summarize that accurately?

Is there any clarity around *how* or *when* the Play Store would initiate such an intent?

Reply:
Yes, that's correct.  Once this change is in a build, we'll have to add `fenix://test_deferred_deep_link` as a deep link in our ongoing Google ad campaign.  That should (hopefully) make Play Store initiate this deep link by trying to start our app with it.  The process should be similar to another app trying to open `fenix://` schemed deep link.

Reply:
One thing to note is I did confirm that if the user is using our app, this particular deep link will not change any of the user experience.  They will not know that they've received a deferred deep link.